vulns/CVE-2024-57996.yml

```diff
@@ -0,0 +1,10 @@
+reachability: Local
+memory_corruption: true
+bug_class: Out of Bounds Write
+impact: LPE
+privileges_required: false
+notes: |-
+  This qdisc could be loaded using unshare -rn and we have a public exploit for
```

Comment:
Is `unshare -rn` enough? The blogpost mentions that vulnerable kernels have `CONFIG_USER_NS`, so I wonder if net namespaces are enough.